Makefile

```diff
@@ -1,3 +1,4 @@
+RUFF_FORMATTED_FOLDERS = sanic examples scripts tests
```

Comment:
Want to add `./guide` while you're at it?

Reply:
@ahopkins thanks for suggestion, this makes sense, though in a spirit of smaller iterations, I would rather prefer to address it separate MR, which I can definitely do.

Reply:
👍 sounds like a plan